langlib/lang.value/src/main/ballerina/value.bal

```diff
@@ -222,6 +274,11 @@ public type JsonFloat ()|boolean|string|float|JsonFloat[]|map<JsonFloat>;
 #
 # Returns an error if the string cannot be parsed.
 #
+# ```ballerina
+# string jsonFloatStr = "12.12";
+# value:JsonFloat jsonFloat = check jsonFloatStr.fromJsonFloatString();
+# ```
```

Comment:
This is not working in Ballerina playground. It may be we need to do some imports. Imo, If so we neeed to add the to this code.

Reply:
The import required is this module itself. So we don't have to add it.

Reply:
why don't we need to add the import statement.
We can add import statement and wrap this with a method.
It may be bit long, but the idea will go to the user completely.
If we didn't add the import statement, user need to find how to import it also.

Reply:
We don't need to wrap it in a function. 

We discussed adding the import briefly when discussing assertions and the conclusion was that the relevant import can be added by whatever mechanism we use for assertion (e.g., as with the Ballerina Shell).

From the user's PoV, I think knowing how to import a module is somewhat of a basic requirement. VSCode code actions will also help. We could add the import here, just that for each example we'll have this import, and copying the snippet (using the copy button) will also copy the import which will not be necessary if the user has already imported the module.